lib/model.js

```diff
@@ -994,7 +994,10 @@ class Model {
         this._defaultValues[name] = _.partial(Utils.toDefaultValue, definition.defaultValue);
       }
 
-      if (definition.hasOwnProperty('unique') && definition.unique !== false) {
+      if (definition.hasOwnProperty('unique') &&
+        definition.unique !== null &&
```

Comment:
May be use https://lodash.com/docs/4.17.4#isEmpty

Reply:
My understanding of the `unique` property is that it can either be a boolean value or a string. As per: http://docs.sequelizejs.com/class/lib/model.js~Model.html#static-method-init

My understanding of `_.isEmpty` is that it tests against objects, collections, maps, or sets. The example from the [lodash docs](https://lodash.com/docs/4.17.4#isEmpty) includes:

```js
_.isEmpty(true);
// => true
```

So, I don't think that `_.isEmpty` would work in this case. 

If we're looking to shorten the code, using a truthy check against null, as in `!=null` would eliminate the need to check for both `undefined` and `null` but it breaks an `eslint` rule, so I chose to be explicit rather than have to add an `eslint` exception.

The cosmetic changes happened because my editor autoformats on save based on the `eslintrc` file, but I agree that it makes the commit noisy. Do you want me to resubmit without any formatting changes?

Reply:
Looks like `_.isNil` may do the trick! I'll use that and resubmit.
https://lodash.com/docs/4.17.4#isNil